app/controllers/reviews_controller.rb

```diff
@@ -118,4 +119,39 @@ def check_review_period
 
     redirect_to root_path, alert: t('reviews.edit.errors.conference_out_of_range')
   end
+
+  def create_third_reject(review)
+    return if in_early_review_phase?
+
+    return unless review.weak_reject? || review.strong_reject?
+
+    session_reviews = review.session.early_reviews + review.session.final_reviews
+    other_reviews = session_reviews.reject { |r| r == review }
+
+    return if other_reviews.count != 1
```

Comment:
We can address Ceci's comment by just making this line:
`return unless review.session.final_reviews.count + 1 == 3`
This makes it more clear that the condition is that there is one missing final review to complete this session's reviews